docgen/src/guide/React_native.md

```diff
@@ -6,9 +6,643 @@ category: guide
 navWeight: 35
 ---
 
-React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/).
+React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/)
+
+In this tutorial, you’ll learn how to:
+
+* install react-instantsearch in your [React Native](https://facebook.github.io/react-native/)  project
+* bootstrap an app using the <InstantSearch> component
+* display infinite results from Algolia
+* add widgets to filter the results
+* connect your own component to the search
+
+## Before we start
+React InstantSearch is meant to be used with Algolia.
+
+Therefore, you’ll need the credentials to an Algolia index. To ease this getting started, here are the credentials to an already configured index:
+
+* appId: latency
+* searchKey: 3d9875e51fbd20c7754e65422f7ce5e1
+* indexName: ikea
```

Comment:
I have seen that on Vue InstantSearch and I actually think it's better when you have already set credentials. If you are following a tutorial you don't want your user to have to fiddle with different browser windows etc.. You want him to follow your text, copy paste, preview, done.

For our tutorials where we make you build a UI reusing a dataset, we should provide default credentials. What I did suggest on Vue is to move the explanation part about those credentials down. Because you don't need to explain this part as early as the begining.

The sooner you get to a result with a tutorial, the better.

I actually think that we should even put all the lines required, like using create-react-app and create-react-native-app (or expo).

The goal of a tutorial is to get up and running as soon as possible without the need to leave the page for another page at any moment.